docs/TESTS.md

```diff
@@ -81,6 +81,16 @@ Good luck! Have fun!
 
    _(Don't worry about the tests failing, at first, this is how you begin each exercise.)_
 
+   If you get the following error:
+
+   ./gradlew: Permission denied
```

Comment:
Could we put this in a code block (using the triple back ticks ```), like the earlier snippets?

Reply:
Sure! Let me just rebase this.

Reply:
Ah, the previous ones had blocks, I didn't see it because I have markup turned off 😅.